handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -1300,34 +1302,34 @@ private static String toJavaCipherSuitePrefix(String protocolVersion) {
     }
 
     @Override
-    public void setUseClientMode(boolean clientMode) {
+    public final void setUseClientMode(boolean clientMode) {
         if (clientMode != this.clientMode) {
             throw new UnsupportedOperationException();
         }
     }
 
     @Override
-    public boolean getUseClientMode() {
+    public final boolean getUseClientMode() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) A getX() method which returns a boolean should be named isX() [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3ABooleanGetMethodName)